portfolio.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```

Comment:
One thing to note on this page is that it doesn't look great on smaller screens.